platform/platform-app/src/main/resources/features.xml

```diff
@@ -167,9 +177,11 @@
         <feature version="${camel.version}">camel-core</feature>
         <feature version="${spring-version-range}">spring</feature>
         <feature version="${spring-version-range}">spring-jms</feature>
-        <bundle dependency="true">mvn:org.apache.geronimo.specs/geronimo-jta_1.1_spec/${geronimo-jta-spec-version}</bundle>
+        <!-- Use version 1.2 of javax transaction, DDF update to the camel feature file -->
+        <feature>transaction</feature>
+        <!-- Use version 2.0 of javax jms, DDF update to the camel feature file -->
+        <bundle dependency="true">mvn:org.apache.geronimo.specs/geronimo-jms_2.0_spec/${geronimo-jms_2_spec.version}</bundle>
```

Comment:
I'm not sure that `geronimo-jms_2_spec.version` should be moved to the `cxf` section of dependencies, it's an `artemis `dependency (used in the broker app and in the camel-sjms platform feature, it looks like)